Makefile

```diff
@@ -78,11 +71,7 @@ ifeq (badgerdb,$(findstring badgerdb,$(COSMOS_BUILD_OPTIONS)))
 endif
 # handle rocksdb
 ifeq (rocksdb,$(findstring rocksdb,$(COSMOS_BUILD_OPTIONS)))
-  ifneq ($(ENABLE_ROCKSDB),true)
-    $(error Cannot use RocksDB backend unless ENABLE_ROCKSDB=true)
-  endif
   CGO_ENABLED=1
-  BUILD_TAGS += rocksdb
```

Comment:
This tag need to come back

Reply:
only if we remove `ENABLE_ROCKSDB`

Reply:
I added back `build_tags += rocksdb` without re-adding `ENABLE_ROCKSDB`